ES/ESmy016.xml

```diff
@@ -31,325 +31,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                 head, which the light of the divinity covers. St Sabbath of Christians,
+                                 repose of the sinners from the torments. Protect me so that I would not
+                                 become a ship of the angel of perdition!</foreign>
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#44r"/><hi rend="rubric">ሰን፡ ክ፡ ቅ፡</hi> ሀብኒ፡ ፍትሐ፡ ሣህልኪ፡
+                              ለላዕለቴ: በከመ፡ ይነሥእ፡ ዓሰባ፡ ፃማሁ፡ ዝንተ፡ አሳባ፡ ድርሳንየ፡ ኢይጥፋዕ፡ በከንቱ።<foreign xml:lang="en">
+                                 <locus target="#44r"/>St Sabbath of Christians, give
+                                 me the justice of your high compassion, as he takes a reward
+                                 of his toils, this the reward of my writing may not perish in
+                                 vain.</foreign>
+                           </explicit>
+                           </msItem>
+                                 </msContents>
                <physDesc>
                   <objectDesc form="Codex">
                      <supportDesc>
                         <support>
-                           <material key="parchment"/>
+                           <material key="parchment">Parchment leaves are poorly cut, with uneven edges.
+                           Cuts or holes are crudely amended on <locus target="#34 #35 #44"/>.</material>
                         </support>
-                        <extent>Made of <measure unit="leaf">44.0</measure> folios, of which
-                              <measure unit="leaf" type="blank"/> blank 42rv, 44v, in <measure unit="quire">6.0</measure> quires . <dimensions type="outer">
+                        <extent>
+                          <measure unit="leaf">44</measure>
+                        <measure unit="leaf" type="blank">3</measure>
+                        <locus target="#42r #42v #44v"/>
+                        <measure unit="quire">6</measure>
+                        <dimensions type="outer">
                               <height unit="mm">145</height>
                               <width unit="mm">110</width>
                               <depth unit="mm">30</depth>
                            </dimensions>
                         </extent>
-                        <foliation><!--2016-10-18+02:00 Updated MY-016 from DomLib--></foliation>
+                        <foliation>Foliation by Ethio-SPaRE, in pencil, in the outer corner of the bottom margin.</foliation>
                         <collation>
                            <list>
-                              <item xml:id="q1">
-                                 <dim unit="leaf">7</dim>
-                                 <locus from="1r" to="8v"/>
+                              <item xml:id="q1" n="1">
+                                 <dim unit="leaf">8</dim>
+                                  <locus from="1r" to="8v"/>
```

Comment:
`<locus from="1r" to="8v"/>`I(8/fols. 1r-8v)
and etc.

Reply:
What do you mean here? I think now it is correct, isnt it?